Gemfile.jruby-3.1.lock.release

```diff
@@ -297,7 +298,7 @@ GEM
       lru_redux (~> 1.1.0)
     logstash-filter-drop (3.0.5)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-elastic_integration (9.0.0-java)
+    logstash-filter-elastic_integration (9.0.1-java)
```

Comment:
I just did a general version bump: 
```
 ./gradlew clean installDefaultGems
 ./vendor/jruby/bin/jruby -S bundle update --all --patch --strict
 cp  Gemfile.lock Gemfile.jruby-3.1.lock.release
```
I can remove the other bumps if desired, but they all look reasonable and low risk. Figured we might as well let them go through?

Reply:
This looks doesn't hurt but for consistency I would like to include [9.1.0](https://github.com/elastic/logstash-filter-elastic_integration/pull/321) next week.